src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -198,11 +198,25 @@
                 }}</span>
               </div>
               <div *ngIf="!isPublic">
-                <span><strong matTooltip="Short description of the workflow">Topic Automatic</strong>:&nbsp;</span>
+                <span
+                  ><strong
+                    matTooltip="Short description of the {{
+                      entryType$ | async
+                    }}, retrieved automatically from the GitHub repository description"
+                    >Topic Automatic</strong
+                  >:&nbsp;</span
+                >
                 <span>{{ workflow?.topicAutomatic }}</span>
               </div>
               <form *ngIf="!isPublic" #editTopic="ngForm" fxLayout="row" fxLayoutAlign="start center" class="w-100">
-                <span><strong matTooltip="Short description of the workflow">Topic Manual</strong>:&nbsp;</span>
+                <span
+                  ><strong
+                    matTooltip="Short description of the {{ entryType$ | async }}, entered manually in the {{
+                      workflow?.mode === WorkflowType.ModeEnum.DOCKSTOREYML ? '.dockstore.yml' : 'UI'
```

Comment:
This message might be a little confusing for `.dockstore.yml` users that have previous entered a topic in the UI, because it'll tell them that it was entered in the `.dockstore.yml`.

Reply:
Interesting you bring that up, I suppose this feature should come with some type of notification to .dockstore.yml users that they should modify their .dockstore.yml to include `topic`s that they might have added in the UI. The webservice PR https://github.com/dockstore/dockstore/pull/5713 won't overwrite existing `topicManual` if they don't do this so we won't lose it, but they should know about it

Reply:
FWIW, I find tooltips are good to have, but half the users don't know to look for them (I often forget myself on other websites), and they shouldn't be relied as the sole way of conveying key info. Could you convey the info some other way as well? Ideas:

* Next to/below the Topic Manual label, have some text saying the text must be edited in the .dockstore.yml.
* When the user clicks the Edit button, a dialog pops saying edit the .dockstore.yml. I'm not 100% wild about this, but this is a brainstorm.

These are suggestions, not blocking, and it could stay the way it is as I suspect most users won't run into this.

Reply:
> FWIW, I find tooltips are good to have, but half the users don't know to look for them (I often forget myself on other websites), and they shouldn't be relied as the sole way of conveying key info

I lean towards keeping it as is because this pattern exists for the other fields on the info tab. If we decide to re-design how to display tooltip info, we can migrate them all at once so the UI is consistent

Reply:
> This message might be a little confusing for .dockstore.yml users that have previous entered a topic in the UI, because it'll tell them that it was entered in the .dockstore.yml.

I specified in the disabled Edit button for GitHub app entries that editing in the UI is deprecated